source/_drafts/Video-js-7-Roadmap.md

```diff
@@ -0,0 +1,67 @@
+In addition to having it be included by default, we’ll make sure to provide builds that exclude VHS for those that don’t need it or are using another playback engine like HLS.js.
+
+## Old IE
+
+![IE8](IE8 small.png)
+
+Video.js has a long history of supporting IE. It has a flash fallback that was originally created for IE8. When Video.js 5 was released, we had a lot of code in place to support IE8 and when Video.js 6 came around, with Flash on its way out, we moved Flash support to a separate project, videojs-flash. Now, based on usage data we have, we are planning to remove support for IE8, IE9, and IE10.
+According to the data we’ve gathered, IE in total has a share of about 4%. Of those 4%, IE8, IE9, and IE10 combined make up barely 5%, and IE11 has around 91% usage. This means that combined IE8, IE9, and IE10 are barely 0.002% of the total usage of Video.js. Based on this incredibly small usage, it is just not worth maintaining support for these browsers. Also, Video.js 6 isn’t going away anytime soon and will be available for your old IE support needs.
+In addition to the statistics, our tests currently take between 5 and 10 minutes to run on IE8. Removing this will allow us to greatly decrease the duration of our test suite. Even worse, sometimes the tests timeout and retry for up to 40 minutes but when they restart they pass on the first try. Not having to deal with these types of issues will allow us to increase our testing infrastructure and provide a better product.
+
+{% pullquote right %}
+![](browserstack logo small.png)
+Thanks [Browserstack](https://www.browserstack.com/) for sponsoring browser-based testing for Video.js!
+{% endpullquote %}
+
+## Video.js 5.x
+
+When Video.js 7 is released it’ll be the end-of-life for Video.js 5.x from a support perspective. Of course, we are not going to unpublish this version from npm or remove the files from the CDN, so you are free to continue using it. However, we urge you to update at your earliest convenience as we will no longer be accepting fixes.
+
+## Google Analytics
+
+{% pullquote right %}
+![](fastly small.png)
+Thanks [Fastly](https://www.fastly.com/) for sponsoring vjs.zencdn.net
+{% endpullquote %}
+
+Video.js provides CDN hosted files for Video.js, sponsored by Fastly -- thanks Fastly! These files currently have a stripped down pixel tracking for Google Analytics (GA) that sends limited information to a GA account we own. We did a terrible job communicating that this is happening and how users can opt out -- set window.HELP_IMPROVE_VIDEOJS to false before loading in Video.js -- or use another CDN like unpkg or CDNJS. Recently we made a change that makes this tracking pixel honor the Do Not Track flag set by users in the browser, however, this isn’t going to affect previously released versions of Video.js
```

Comment:
Should we make the first or second Fastly inline a link as well?

Reply:
Sure